### DISCUSSION TITLE
Security Patch Version - Question around Patched Releases

### DISCUSSION THREAD

**Discussion by ncphins** [keycloak/keycloak#34048] [Q&A] [answered]
I am not sure this is the place for this, but I wanted to check. This is sort of a general question I have had for a while.

I see a patch is available for a security issue in the Security area. I see that the patch info lists 24.0.7 as the patched version, however I do not see a 24.0.7 release. And I am guessing there will not be. Or I am looking in the wrong place.

I need to patch the vulnerability, but I prefer not to go to 25 unless needed. Can someone help me understand? Will there be a 24.0.7 release?

Here is an example of an issue I am looking at: https://github.com/keycloak/keycloak/security/advisories/GHSA-5rxp-2rhr-qwqv

**Comment by stianst** [comment, marked as answer]
We only support and release the latest release, which is 26 at the moment.

Red Hat provides longer support for some releases (22 and 24 are both still in support by Red Hat). They make the source code and tags available in the upstream, but we don't do any release from that.

**Reply by ncphins** [reply]
Thanks! I figured it was something like that. I appreciate the clarification and we can plan accordingly. It has confused me in the past and I should have asked then.
